CarRentalService.Tests/CarRentalServiceTests.cs

```diff
@@ -0,0 +1,134 @@
+using CarRentalService.Domain.Domain;
+using CarRentalService.Tests.Fixture;
+
+namespace CarRentalService.Tests;
+
+/// <summary>
+/// Unit tests for CarRentalService rental operations and reporting
+/// </summary>
+public class RentalServiceTests : IClassFixture<CarRentalFixture>
+{
+    private readonly CarRentalFixture _fixture;
+
+    /// <summary>
+    /// Initializes a new instance with the provided test data fixture
+    /// </summary>
+    /// <param name="fixture">The test data fixture containing rental service data</param>
+    public RentalServiceTests(CarRentalFixture fixture)
+    {
+        _fixture = fixture;
+    }
+
+    /// <summary>
+    /// Retrieves information about all customers who rented cars of the specified model, ordered by full name
+    /// </summary>
+    [Fact]
+    public void GetCustomersRentingModel_OrderedByName()
```

Comment:
Именуем методы в PascalCase тут и ниже по коду
```suggestion
    public void GetCustomersRentingModelOrderedByName()
```